schemas/stsci.edu/asdf/transform/label_mapper-0.1.0.yaml

```diff
@@ -0,0 +1,138 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/label_mapper-0.1.0"
+tag: "tag:stsci.edu:asdf/transform/label_mapper-0.1.0"
+title: >
+  Represents a _LabelMapper.
+description: |
+  A label mapper instance maps locations to a label.
+  It is used together with RegionsSelector. The LabelMapper
+  returns the label corresponding to given inputs. The RegionsSelector
+  returns the transform corresponding to this label. This maps uniquely inputs
+  (e.g. pixels on a detector) to transforms.
+
+examples:
+  -
+    - |
+        A LabelMapperArray - array indices are mapped to labels.
```

Comment:
I don't know if it makes sense to include Python examples here -- the ASDF spec should probably be generic.  I'd just say: "Map array indices to labels" and not have the Python code snippet.